web/src/test/java/log4j.xml

```diff
@@ -1,25 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE log4j:configuration SYSTEM "log4j.dtd">
-<log4j:configuration>
```

Comment:
Dropwizard uses Logback, not log4j -- I hope it as reasonable defaults (I should think so). This file was in a test directory anyway -- can't do anything in production.